### DISCUSSION TITLE
Schedule for enrolling GitHub Copilot Labs features to JetBrains IDE plugin

### DISCUSSION THREAD

**Discussion by tracer0tong** [community/community#52141] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Is there any plan or schedule to offer additional features from GitHub Copilot Labs in JetBrains IDE? I'm using the business version of Copilot, and the main IDEs for me are JetBrains PyCharm, and Goland. However, there are no Copilot Labs available for these IDEs, this is quite sad.

**Comment by Akash1134** [comment, marked as answer]
Hi @tracer0tong ,

Copilot Labs is currently a usable prototype with VS Code extension for experimental applications of GitHub Copilot and it will remain experimental in nature for the forcible future.

It is hard to comment when we'll have a schedule for enrolling into JetBrains IDE plugin.

[Copilot Labs](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-labs) is intended to be experimental in nature, so things might break occasionally. We want to share early features with you to learn what works, what doesn’t, and incorporate your feedback. Labs experiments may or may not graduate into permanent features of GitHub Copilot.

There is no waitlist but you will need to accept the [GitHub Next Pre-Release License Terms](https://github.com/githubnext/githubnext/blob/main/TERMS_AND_CONDITIONS.md).

You will also need access to GitHub Copilot itself. You can sign up at https://github.com/features/copilot/.

Then install both the [GitHub Copilot extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) and the [GitHub Copilot Labs extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-labs) for VS Code. GitHub Copilot Labs works only with VS Code right now.

**Reply by tracer0tong** [reply]
Thanks for reply, I have Copilot subscription, but I don't think I'm ready to change IDE.
Summarizing your answer: no plans to support Lab features in JetBrains IDE.
